lib/config/config-validator.js

```diff
@@ -141,20 +144,26 @@ function validateRuleOptions(rule, ruleId, options, source = null) {
 /**
  * Validates an environment object
  * @param {Object} environment The environment config object to validate.
- * @param {Environments} envContext Env context
  * @param {string} source The name of the configuration source to report in any errors.
+ * @param {function(envId:string): Object} [getAdditionalEnv] A map from strings to loaded environments.
  * @returns {void}
  */
-function validateEnvironment(environment, envContext, source = null) {
+function validateEnvironment(
+    environment,
+    source,
+    getAdditionalEnv = Function.prototype
```

Comment:
Can we use something like

```js
const noop = Function.prototype;

function validateEnvironment(
    environment,
    source,
    getAdditionalEnv = noop
) {}
```

instead? I think the usage of `Function.prototype` directly as a noop function might be confusing without further explanation.